### ISSUE TITLE
Blend Shapes visual bug on Linux and video-linux on AMD gpu

### ISSUE THREAD

**Issue by Nanox19435** [godotengine/godot#41688]
**Godot version:**

3.2.2 stable

**OS/device including version:**
 5.7.17-2-MANJARO

AMD Radeon RX 5600 XT, video-linux, GLES3

**Issue description:**

I used VRoid studio to create some anime style 3d models, then imported them to Godot, but when I change the shape keys, the model deforms creating this weird effect of vertex displacement. Sometimes also the 3d editor move arrows glitch as well, as you can see on the second picture.

The project will work fine on windows, and has also run as expected on an intel graphics machine in both windows and Linux, so the issue lies on the video drivers, as far as I can tell.

**Steps to reproduce:**
Import as glb a vrm file (generated with VRoid Studio), you only need to change the extension from .vrm to .glb to do this.
Create a new inherited scene from the imported asset
Select the MeshInstance node named Face
On the inspector open the dropdown menu of Blend Shapes
Change any Blend Shape and the bug will appear

**Minimal reproduction project:**

![image](https://user-images.githubusercontent.com/50684926/91900984-21bb5a80-ec65-11ea-8e2d-d24e36ecece6.png)
![image](https://user-images.githubusercontent.com/50684926/91901914-80350880-ec66-11ea-8dee-7c81a78342d9.png)

**Comment by fire** [comment]
This looks very similar to https://github.com/KhronosGroup/glTF-Blender-IO/issues/308.

**Comment by Nanox19435** [comment]
After more testing using more systems, I was able to pin point the blame here. This is a problem with the AMD drivers for my graphics card (5600 XT) on linux, not godot (as far a i can tell). I have been able to "correct" this problem by changing on Manjaro settings the drivers from video-linux to video-vesa, but doing that prevents you from using almost any resolution on your screen. I hope that an update on the drivers corrects this in the future.
